//公会解散
message DceGuildDismiss
{
}